src/main/java/com/epam/spring/hw3/repository/impl/OrderRepositoryImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.spring.hw3.repository.impl;
+
+import com.epam.spring.hw3.model.Order;
+import com.epam.spring.hw3.repository.OrderRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Component

```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers